canvas_modules/harness/webpack.config.dev.js

```diff
@@ -141,4 +141,4 @@ module.exports = {
 		rules: rules
 	},
 	plugins: plugins
-};
+};
```

Comment:
Why are these files changed?  Looks like an addition line?

Reply:
For the webpack.config.dev.js and webpack.config.prod.js, I reverted them to their original state because I believe I added a comma in the previous commit last Thursday.

Reply:
These are still showing up as changed though.